plugin/runmetrics/producer.go

```diff
@@ -219,6 +229,52 @@ func newMemStats(producer *producer) (*memStats, error) {
 		return nil, err
 	}
 
+	// GC
+	memStats.gcSys, err = producer.createInt64GaugeEntry("process/gc_sys", "Bytes of memory in garbage collection metadats", metricdata.UnitBytes)
+	if err != nil {
+		return nil, err
+	}
+
+	memStats.otherSys, err = producer.createInt64GaugeEntry("process/other_sys", "Bytes of memory in miscellaneous off-heap runtime allocations", metricdata.UnitBytes)
+	if err != nil {
+		return nil, err
+	}
+
+	memStats.numGC, err = producer.createInt64GaugeEntry("process/num_gc", "Cumulative count of completed GC cycles", metricdata.UnitDimensionless)
+	if err != nil {
+		return nil, err
+	}
+
+	memStats.numForcedGC, err = producer.createInt64GaugeEntry("process/num_forced_gc", "Cumulative count of GC cycles forced by the application", metricdata.UnitDimensionless)
+	if err != nil {
+		return nil, err
+	}
+
+	memStats.nextGC, err = producer.createInt64GaugeEntry("process/next_gc", "Target heap size of the next GC cycle in bytes", metricdata.UnitBytes)
+	if err != nil {
+		return nil, err
+	}
+
+	memStats.lastGC, err = producer.createInt64GaugeEntry("process/last_gc", "Time the last garbage collection finished, as milliseconds since 1970 (the UNIX epoch)", metricdata.UnitMilliseconds)
+	if err != nil {
+		return nil, err
+	}
+
+	memStats.pauseTotalNs, err = producer.createInt64GaugeEntry("process/pause_total", "Cumulative milliseconds spent in GC stop-the-world pauses", metricdata.UnitMilliseconds)
+	if err != nil {
+		return nil, err
+	}
+
+	memStats.pauseTotalNs, err = producer.createInt64GaugeEntry("process/pause_total", "Cumulative milliseconds spent in GC stop-the-world pauses", metricdata.UnitMilliseconds)
```

Comment:
duplicate. Same as line 263.

Reply:
All cumulative values can use [AddInt64Cumulative](https://github.com/census-instrumentation/opencensus-go/blob/df6e2001952312404b06f5f6f03fcb4aec1648e5/metric/registry.go#L184)

this allows to compute rate on certain backends.

Reply:
The problem with AddInt64Cumulative is that it expects an incremental value, which is not how this metric can be recorded. These would be similar to how `memMalloc` and `memFrees` work in this producer in that the runtime is incrementing for you and all that you can record is the total value.

Reply:
In that case you could use [AddInt64DerivedCumulative](https://github.com/census-instrumentation/opencensus-go/blob/643eada29081047b355cfaa1ceb9bc307a10423c/metric/test/exporter_test.go#L68).

Reply:
I finally did this, I also did this retroactively for the metrics that I believe are also cumulative in this runmetrics package since it seems weird to do this only for net new metrics.

Reply:
Sorry for the long delay. The problem with retroactively changing the type for existing metrics is that it will fail for some backends. For example, stackdriver will reject the same metric if the type is different. But on the other hand it is the right thing to do. We'll have add release note as breaking change for this plugin.
Alternatively, it could support both options. Provide a flag to choose the option.

Reply:
Makes sense, I'll add a flag for this to ensure people can migrate at their own pace.

Reply:
However, should this flag control only the ones necessary to stay passive or all cumulative metrics? I lean towards the latter but I can understand why someone would want the former.

Reply:
Not really happy with this but honestly rather than have a bunch of `if` statements running around it made sense to have separate structures.